src/components/accession2/Accession2Address.tsx

```diff
@@ -70,7 +71,7 @@ export default function Accession2Address(props: Accession2AddressProps): JSX.El
     }
   };
 
-  if (!isOpen) {
+  if (!isOpen && !opened) {
```

Comment:
isOpen captures opened state of component.. we could instead change line 20 to
```
const [isOpen, setIsOpen] = useState<boolean>(opened || false);
```
Same thing in other places where we have `opened`.